ui/app/src/components/SitesGrid/SiteCard.tsx

```diff
@@ -102,7 +102,7 @@ export default function SiteCard(props: SiteCardProps) {
           <CardMedia
             component="img"
             className={classes.media}
-            image={`/.crafter/screenshots/default.png?crafterSite=${site.id}`}
+            image={`/.crafter/screenshots/default.png?crafterSite=${site.id}&v=${Date.now()}`}
```

Comment:
We agreed on using some sort of count/state. This forces re-painting image on every render.